docs/src/main/java/io/helidon/docs/se/guides/Upgrade4xSnippets.java

```diff
@@ -170,4 +170,46 @@ void snippet_8() {
         Config config = Config.global();
         // end::snippet_8[]
     }
+
+    /* Helidon 3 code
+    static Single<WebServer> startServer() {
+        Config config = Config.create();
+
+        WebServer server = WebServer.builder(createRouting(config))
+                .config(config.get("server"))
+                .addMediaSupport(JsonpSupport.create())
+                .build();
+
+        // tag::snippet_9[]
+        Single<WebServer> webserver = server.start();
+
+        webserver.thenAccept(ws -> {
+                    System.out.println("WEB server is up! http://localhost:" + ws.port() + "/greet");
+                    ws.whenShutdown().thenRun(() -> System.out.println("Helidon WebServer has stopped"));
+                })
+                .exceptionallyAccept(t -> {
+                    System.err.println("Startup failed: " + t.getMessage());
+                    t.printStackTrace(System.err);
+                });
+        // end::snippet_9[]
+        return webserver;
+    }
+     */
```

Comment:
I see the purpose of including the v3 code here, but commenting out (part of) the code in a snippet pretty much defeats the purpose of it being a snippet--making sure it compiles, uses valid APIs, etc.

If we decide to _not_ show the v3 code inline in the v4 update doc then my comment becomes moot.

Reply:
See my previous comment. I am following the pattern already used in the upgrade guide.